mc/mc.go

```diff
@@ -206,6 +202,13 @@ func MultiCurl(ctx context.Context, cfg *Config) (int, ResultStats) {
 		case <-time.After(cfg.RepeatDelay):
 			// normal pause
 		}
+		if cfg.ReLookup && cfg.IPFile == "" {
+			log.LogVf("Re-resolving %s host %s", cfg.ResolveType, cfg.host)
+			addrs, err = Resolve(ctx, cfg)
+			if err != nil {
+				return log.FErrf("Unable to re-resolve %s host %s: %v", cfg.ResolveType, cfg.host, err), result
```

Comment:
this can't really happen (succeeds first time then fails, so it decreases the coverage to have that line in this PR)